tests/test_browser.py

```diff
@@ -2582,3 +2582,93 @@ def test_modularize(self):
         ''' % code)
         self.run_browser('a.html', '...', '/report_result?0')
 
+  # Test that the emscripten_ atomics api functions work.
```

Comment:
Hmm, you could avoid adding anything to `src/` by having an `onerror` handler in the tests, and if the handler sees there are no SharedArrayViews, it reports a result (of pass or warn).